Description: Upstream changes introduced in version 0.33~cvs20110918-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 dietlibc (0.33~cvs20110918-2) unstable; urgency=low
 .
   * d/patches/fix-ppc64.patch: Fix FTBFS (Closes: #644405)
     - Thanks Hiroyuki Yamamoto for the fix
 .
 The person named in the Author field signed this changelog entry.
Author: Hector Oron <zumbi@debian.org>
Bug-Debian: http://bugs.debian.org/644405

---

Index: dietlibc-0.33~cvs20111108/ppc64/start.S
===================================================================
--- dietlibc-0.33~cvs20111108.orig/ppc64/start.S	2011-11-08 21:23:39.000000000 +0000
+++ dietlibc-0.33~cvs20111108/ppc64/start.S	2011-11-08 21:24:08.000000000 +0000
@@ -78,7 +78,7 @@
 	.ascii "_start"
 .LT_start_name_end:
 	.align 2
-	.size _start,.-_start
+	.size ._start,.-._start
 
 /* Define a symbol for the first piece of initialized data.  */
 	.section ".data"
